server/presenters/rest/DataConverterREST.ts

```diff
@@ -0,0 +1,12 @@
+import OutputData from "../../output-data/SessionOutputData";
+
+export default class DataConverterREST {
```

Comment:
Konverterius mes vadinam taip: GameD2BConverter
Game - modelis kurį konvertuoji
D2B - Domain 2(to) Boundary, dar gali būti R2B(rest2boundary), B2R ir B2D